frontend/src/views/ProprietairePage.vue

```diff
@@ -803,13 +806,17 @@ export default defineComponent({
               <DsfrInputGroup
                 :is-valid="isNomSivValid"
                 :error-message="nomSivErrorMessage"
+                description-id="nom-particulier-SIV-erreur-message"
               >
                 <DsfrInput
                   v-model="formData.siv.titulaire.particulier.nom"
                   label="Nom de naissance"
                   label-visible
                   hint="Tel qu'indiqué sur le certificat d'immatriculation."
                   required
+                  aria-required="true"
+                  :aria-invalid="!isNomSivValid"
+                  aria-errormessage="nom-particulier-SIV-erreur-message"
```

Comment:
Pour tous les champs input, ajouter l'attribut aria-describedby avec la même valeur que pour aria-errormessage

```suggestion
                  aria-errormessage="nom-particulier-SIV-erreur-message"
                  aria-describedby="nom-particulier-SIV-erreur-message"
```